interview-questions.md

```diff
@@ -8,50 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: 
 
 Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git stands for Global Information Tracker and gives you a way to access information remotely or upload remotely to a "hub" of some sort.  Github is a place where you can use git to send files documents and data to a central hub to be accesses.
 
-Researched answer:
+Git is the tool that lets you move things around such as files and documents and Github is the place to hold said files or to pull files and data from.
+
+Researched answer: Git is an open source code management that helps companies and people track changes in code and actually helps multiple web developers to work on projects together. Git is a way to track changes in your code while Github is a cloud-based service that lets you manage and use git repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The javascript operators that will return a Boolean value are AND (&&), OR (||), and NOT (!). 
 
-Researched answer:
+Researched answer: After reasearching I keep coming up with the same answer that the JavaScript operators that will return a Boolean value are, AND (&&), OR(||), and NOT (!).
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: Index is the location of a value starting from 0.  The difference between an index and a value is that an index is the location of a value while the value is the type of data at that index.
 
-Researched answer:
+Researched answer: Index is the position of a value in code, and the difference between index and value is that the index is the position at which a specific value is located. 
 
 4. What is iteration?
 
-Your answer:
+Your answer: Iteration is doing a certain thing multiple times until it is told to stop.
 
-Researched answer:
+Researched answer: Iteration is a essentially a loop through a function or program that will keep going through it until it meets the requirement not to repeat itself.
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-Your answer:
+Your answer: For proper pair programming you have two roles.  One is the driver who has hands on the keyboard typing the code and using the text editor.  While the other role is navigator who is directing the driver by telling him what code to write or looking at the code already written and focusing entirely on that.  
 
-Researched answer:
+Researched answer: The driver is behind the keyboard typing the coding while the navigator is focusing on the overal direction of the coding.  They help each other by discussing difficult code or solving issues together.
 
 ## Looking Ahead: Terms for Next Week
 
```

Comment:
It's always great to look ahead at the terms so you have some context when hearing the terms during lectures.